test/api/v3/integration/user/auth/POST-register_local.test.js

```diff
@@ -52,6 +53,22 @@ describe('POST /user/auth/local/register', () => {
       expect(user.tasksOrder.habits).to.have.a.lengthOf(0);
     });
 
+    it('enrolls new users in an A/B test', async () => {
+      let username = generateRandomUserName();
+      let email = `${username}@example.com`;
+      let password = 'password';
+
+      let user = await api.post('/user/auth/local/register', {
+        username,
+        email,
+        password,
+        confirmPassword: password,
+      });
+
+      await expect(getProperty('users', user._id, '_ABtest')).to.eventually.exist;
```

Comment:
Actually, this is the only thing I'd change, so I'll just make the change and merge it in. (two async calls means it takes twice as long, which is why I'd keep it at one async call)